Resources/Prototypes/_Goobstation/Changeling/Entities/Objects/Weapons/Melee/changeling_armblade.yml

```
@@ -0,0 +1,50 @@
+  - type: Tool
+    qualities:
+    - Slicing
+    - Prying
+  - type: DisarmMalus
+    malus: 0
+
+- type: entity
+  parent: ArmBladeChangeling
+  id: FakeArmBladeChangeling
+  components:
+  - type: MeleeWeapon
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)